src/agentTestCache.ts

```diff
@@ -59,7 +63,7 @@ export class AgentTestCache extends TTLConfig<TTLConfig.Options, CacheContents>
     }
 
     if (runId) {
-      return { runId };
+      return this.get(runId);
```

Comment:
the current behavior allows to resume from `--job-id` even if it's not in the cache, I think this will break line change will break that:

![Screenshot 2025-02-28 at 11 25 31](https://github.com/user-attachments/assets/f8b1e0ad-fac1-42ae-a71a-fc04041757a4)